src/services/results-calculator.ts

```diff
@@ -6,7 +6,9 @@ import {
 
 export default class ResultsCalculator {
 
-	results: { [key: string]: number } = {}
+	results: { 
+		[key: string]: number 
+	} = {}
 
 	parseLine (line: string) {
```

Comment:
These 'parse' methods do not seem to be doing any calculations of the results. It may be more appropriate to create another class whose purpose is the parse the lines from the input text and return values. These values would then be passed to the methods in ResultsCalculator to actually calculate scores.

Reply:
The 'parseScore' method will only work if the score is a single digit (e.g. it will work if a team scored 2 but will not work if they scored 12 or 120). Perhaps doing a read from right to left, stopping at the first space, and then doing a try / catch to convert to a Number would do the trick.